src/Transactions/TransactionDetails/TransactionInformation/TransactionInformation.js

```diff
@@ -84,7 +84,7 @@ const TransactionInformation = ({
         >
           <KeyValue
             label={<FormattedMessage id="ui-finance.transaction.type" />}
-            value={transactionType}
+            value={<FormattedMessage id={`ui-finance.transaction.type.${transactionType}`} />}
```

Comment:
IMHO, it might be a good idea to add a default message with transactionType in case something new is added on back-end and there is no corresponding change on UI

Reply:
A good idea for the mentioned case